docs/flowcollector-flows-netobserv-io-v1alpha1.adoc

```diff
@@ -0,0 +1,1327 @@
+// Automatically generated by 'openshift-apidocs-gen'. Do not edit.
+:_content-type: ASSEMBLY
+[id="flowcollector-flows-netobserv-io-v1alpha1"]
+= FlowCollector [flows.netobserv.io/v1alpha1]
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+Description::
++
+--
+FlowCollector is the Schema for the flowcollectors API, which pilots and configures netflow collection.
+--
+
+Type::
+  `object`
+
+
+
+== Specification
+
+[cols="1,1,1",options="header"]
+|===
+| Property | Type | Description
+
+| `apiVersion`
+| `string`
+| APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+
+| `kind`
+| `string`
+| Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+
+| `metadata`
+| xref:../objects/index.adoc#io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta[`ObjectMeta`]
```

Comment:
The xref:../ was breaking my build. We can't put xrefs in modules, sadly its a limitation of our docs. I've just put `ObjectMeta` in the product documentation

Reply:
ok - when automating, we will take care of removing it